react-site/src/pages/ResultsPage/ResultsPage.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import "./ResultsPage.css";
+
+import Card from "../../components/Card";
+import Heading from "../../components/Heading";
+import Placeholder from "../../components/Placeholder";
+import Text from "../../components/Text";
+
+const FEATUREDLENGTH = 4;
+
+export const ResultsPage = ({ query, data, ready, setSelectedObject }) => {
+  return (
+    <>
+      <Heading variant="h1">Looking for &ldquo;{query}&rdquo;</Heading>
+      <div className="results">
+        <div className="featured">
+          {!ready
+            ? Array.from(Array(FEATUREDLENGTH)).map((_, i) => {
+                return (
+                  <div key={i} className="result">
+                    <Placeholder
+                      style={{
+                        height: "128px",
+                        width: "100%",
+                      }}
+                    />
+                  </div>
```

Comment:
The cards look nice, but how the results are structured in two columns are a little confusing. It might be more intuitive if the results are numbered based on their indices, and "More results" could show up in a smaller font or be removed.

Reply:
makes sense, will make more results smaller for now. not sure on the numbering, but we can look into that as well